examples/DancingGoat/DancingGoat.csproj

```diff
@@ -1,11 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
+    <TargetFramework Condition="'$(TargetFramework)' == ''">net8.0</TargetFramework>
     <RootNamespace>DancingGoat</RootNamespace>
-    <UserSecretsId>79c858af-5444-4954-8d08-6592a961e82a</UserSecretsId>
+    <UserSecretsId>ecf97e0b-2de4-4b70-a247-e9e3ab70b113</UserSecretsId>
```

Comment:
Switching the GUID because all Dancing Goat projects have the same user secrets GUID which can cause problems.